Makefile

```
@@ -33,7 +33,7 @@ test:
 
 .PHONY: testacc
 testacc:
-	TF_ACC=1 $(TEST_CMD) ./...
+	TF_ACC=1 $(TEST_CMD) ./internal/provider/resources/emailtemplate_test.go # ./...
```

```suggestion
	TF_ACC=1 $(TEST_CMD) ./...
```

btw, you can run a subset of these yourself like so:
```
TF_ACC=1 go test ./internal/provider/resources/emailtemplate_test.go
```